apps/token/src/routes/governance/propose/network-parameter/propose-network-parameter.tsx

```diff
@@ -0,0 +1,292 @@
+              minProposerBalance={minProposerBalance}
+              spamProtectionMin={minSpamBalance}
+            />
+
+            {VEGA_DOCS_URL && (
+              <div className="text-sm">
+                <ProposalDocsLink
+                  urlPart1={VEGA_DOCS_URL}
+                  urlPart2={'/tutorials/proposals/network-parameter-proposal'}
+                />
+              </div>
+            )}
+
+            {VEGA_EXPLORER_URL && (
+              <p className="text-sm">
+                {t('MoreNetParamsInfo')}{' '}
+                <Link
+                  href={`${VEGA_EXPLORER_URL}/network-parameters`}
+                  target="_blank"
+                >{`${VEGA_EXPLORER_URL}/network-parameters`}</Link>
+              </p>
+            )}
+
+            <div data-testid="network-parameter-proposal-form">
+              <form onSubmit={handleSubmit(onSubmit)}>
+                <ProposalFormSubheader>
+                  {t('ProposalRationale')}
+                </ProposalFormSubheader>
+
+                <ProposalFormTitle
+                  registerField={register('proposalTitle', {
+                    required: t('Required'),
+                  })}
+                  errorMessage={errors?.proposalTitle?.message}
+                />
+
+                <ProposalFormDescription
+                  registerField={register('proposalDescription', {
+                    required: t('Required'),
+                  })}
+                  errorMessage={errors?.proposalDescription?.message}
+                />
+
+                <ProposalFormSubheader>
+                  {t('SelectAParameterToChange')}
+                </ProposalFormSubheader>
+
+                <FormGroup
+                  label={t('SelectAParameterToChange')}
+                  labelFor="proposal-parameter-key"
+                  hideLabel={true}
+                >
+                  <Select
+                    data-testid="proposal-parameter-select"
+                    id="proposal-parameter-key"
+                    {...register('proposalNetworkParameterKey', {
+                      required: t('Required'),
+                    })}
+                    onChange={(e) => setSelectedNetworkParam(e.target.value)}
+                    value={selectedNetworkParam}
+                  >
+                    <option value="">{t('SelectParameter')}</option>
+                    {networkParamsData?.map(({ key }) => (
+                      <option key={key} value={key}>
+                        {key}
+                      </option>
+                    ))}
+                  </Select>
+                  {errors?.proposalNetworkParameterKey?.message && (
+                    <InputError intent="danger">
+                      {errors?.proposalNetworkParameterKey?.message}
+                    </InputError>
+                  )}
+                </FormGroup>
+
+                {selectedNetworkParam && (
+                  <div className="mt-[-10px]">
```

Comment:
chore (non blocking): can't we just rely on the default form group spacing?

Reply:
Default form group spacing correctly provides default spacing for elements so they're separated from one another. These need to be closer to indicate they're entwined